collect_app/src/main/AndroidManifest.xml

```diff
@@ -140,9 +140,9 @@ the specific language governing permissions and limitations under the License.
             android:name=".gdrive.GoogleSheetsUploaderActivity"
             android:configChanges="orientation|screenSize" />
         <activity
-            android:name=".activities.InstanceUploaderListActivity"
+            android:name=".readytosend.InstanceUploaderListActivity"
```

Comment:
Could the package just be named `send`? Or maybe live inside `formmanagement` and then be `.formmanagement.send`?

Reply:
Ok I renamed it to send but moved to `instancesmanagement` instead of `formmanagement`